apps/fluent-tester/src/E2E/README.md

```diff
@@ -0,0 +1,77 @@
+# E2E Testing Overview
+
+## Prerequisites
```

Comment:
Do the pre-reqs matter in the context of the repo? Besides the developer mode step, which I would probably add to the Running E2E Tests section

Reply:
That's to say all of these things are dependencies of the repo itself, right?

Reply:
You're correct for some. For the Node.js part, the version needs to be above 10.19, so I thought that important to add. I could probably remove the React-Native windows dependencies, but I thought I would add it because it's especially important for UWP.

However, WinAppDriver is not a dependency of the repo itself, you actually need to download this on your machine.

Reply:
Do we warn or error cleanly if WinAppDriver needs to be installed and isn't?  Seemingly it should go in devDependencies since it's a development dependency

Reply:
ah I see, the binary probably isn't being packaged into any npm package that we can pull in as a devDependency. Did you have to do anything with the CI loops to make sure that WinAppDriver was installed, or does the windows-2019 image just come with it. Also echoing Pat's point, I would make sure we get an intelligible error message in the uninstalled case.

Reply:
windows-2019 image comes with it. 

And when running the E2E tests without WinAppDriver, Appium returns an Error specifically calling out that WinAppDriver isn't installed, and it fails. Is that sufficient?

Reply:
Sounds sufficient to me